shape_msgs/SolidPrimitive primitive
string primitive_id 
---
bool success
